common/lib/xmodule/xmodule/templates/problem/drag_and_drop.yaml

```diff
@@ -5,21 +5,22 @@ metadata:
     showanswer: never
 data: |
     <problem>
+      <p>In drag and drop problems, students respond to a question by dragging text or objects to a specific location on an image.</p>
       <p>
-      In drag and drop problems, students respond to a question by dragging text or objects to a specific location on an image.
+          For more information, see
+          <a href="http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/drag_and_drop_deprecated.html" target="_blank">
+          Drag and Drop Problem (Deprecated)</a> in <i>Building and Running an edX Course</i>.
       </p>
       <p>
-      For more information, see
-      <a href="http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/drag_and_drop_deprecated.html" target="_blank">
-      Drag and Drop Problem (Deprecated)</a> in <i>Building and Running an edX Course</i>.</p>
-      <p>When you add the problem, be sure to select <strong>Settings</strong>
-        to specify a <strong>Display Name</strong> and other values that apply.</p>
+          When you add the problem, be sure to select <strong>Settings</strong>
+          to specify a <strong>Display Name</strong> and other values that apply.
+      </p>
       <p>You can use the following example problems as models.</p>
-      <hr />
-       <customresponse>
-         <h3>Simple Drag and Drop</h3>
-        <p>Drag each word in the scrollbar to the bucket that matches the number of letters in the word.</p>
-                <drag_and_drop_input img="https://studio.edx.org/c4x/edX/DemoX/asset/L9_buckets.png">
+      <hr/>
+      <customresponse>
+          <h3>Simple Drag and Drop</h3>
+          <p>Drag each word in the scrollbar to the bucket that matches the number of letters in the word.</p>
```

Comment:
@lamagnifica @muhammad-ammar I just came to know that we won't be making any changes to the Drag & Drop problems since it's deprecated and open craft is working on a new component for this. @sstack22 right?